model/src/main/java/org/cbioportal/model/Sample.java

```diff
@@ -119,11 +120,13 @@ public void setSequenced(Boolean sequenced) {
         this.sequenced = sequenced;
     }
 
-    public Boolean getCopyNumberSegmentPresent() {
-        return copyNumberSegmentPresent;
-    }
+    public Boolean getCopyNumberSegmentPresent() { return copyNumberSegmentPresent; }
 
     public void setCopyNumberSegmentPresent(Boolean copyNumberSegmentPresent) {
         this.copyNumberSegmentPresent = copyNumberSegmentPresent;
     }
+
+    public Boolean getFusionPresent() { return fusionPresent; }
+
+    public void setFusionPresent(Boolean fusionPresent) { this.fusionPresent = fusionPresent; }
```

Comment:
Can you format these methods the same was as the existing methods in this class?

Reply:
done.